go/base/static/js/test/routing/views.test.js

```diff
@@ -118,53 +118,111 @@ describe("go.routing (views)", function() {
       setUp();
       diagram = newRoutingDiagram();
 
-      var model = new ChannelModel({
-        uuid: 'channel4',
-        tag: ['invisible_sms', '*181#'],
-        name: '*181#',
-        description: 'Invisible Sms: *181#',
-        endpoints: [
-          {uuid: 'endpoint80', name: 'default'},
-          {uuid: 'endpoint81', name: 'secondary'}]
+      var model = new RouterModel({
+        uuid: 'router3',
+        type: 'keyword',
+        name: 'keyword-router',
+        description: 'Keyword',
+        channel_endpoints: [{uuid: 'endpoint12', name: 'default'}],
+        conversation_endpoints: [
+          {uuid: 'endpoint13', name: 'default'},
+          {uuid: 'endpoint14', name: 'default'},
+          {uuid: 'endpoint15', name: 'default'},
+          {uuid: 'endpoint16', name: 'default'}
+        ]
       });
 
-      state = diagram.states.add('channels', new RoutingStateView({
+      state = diagram.states.add('routers', new RouterStateView({
         model: model,
         diagram: diagram,
-        collection: diagram.states.members.get('channels')
+        collection: diagram.states.members.get('routers')
       }), {render: false, silent: true});
+
     });
 
     afterEach(function() {
       tearDown();
     });
 
+    describe(".endpointsForSide", function() {
+      it("should return the endpoints for a side", function() {
+        assert.deepEqual(
+          state.endpointsForSide('left'),
+          [state.endpoints.get('endpoint12')]
+        );
+        assert.deepEqual(state.endpointsForSide('right'), [
+          state.endpoints.get('endpoint13'),
+          state.endpoints.get('endpoint14'),
+          state.endpoints.get('endpoint15'),
+          state.endpoints.get('endpoint16')
+        ]);
+      });
+    });
+
+    describe(".findMaxEndpoints", function() {
+      it("should return the max of two endpoint arrays", function() {
+        assert.equal(state.findMaxEndpoints(), 4);
+      });
+    });
+
+    describe(".tooManyEndpoints", function() {
+      it("should check if there are too many endpoints", function() {
+        assert.equal(state.tooManyEndpoints(), true);
```

Comment:
`<bikeshed>`Could we use `assert()` here since its a boolean check?`</bikeshed>`